addons/hr_recruitment/wizard/applicant_refuse_reason.py

```diff
@@ -1,14 +1,105 @@
 # -*- coding: utf-8 -*-
 
-from odoo import api, fields, models
+from odoo import api, fields, models, _
+from odoo.tools.misc import get_lang
 
 
 class ApplicantGetRefuseReason(models.TransientModel):
     _name = 'applicant.get.refuse.reason'
+    _inherits = {'mail.compose.message': 'composer_id'}
     _description = 'Get Refuse Reason'
 
+    @api.model
+    def default_get(self, fields):
+        res = super(ApplicantGetRefuseReason, self).default_get(fields)
+        res_ids = self._context.get('default_applicant_ids')
+
+        composer = self.env['mail.compose.message'].create({
```

Comment:
Don't do that. Have computed fields for recipients and correctly have subject / body rendering content coming from template. There are several examples of lightweight code doing that kind of notification wizard in Odoo, I let you dive in it :) .